src/strategies/chubbykaijudao/index.ts

```diff
@@ -0,0 +1,49 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { Multicaller } from '../../utils';
+
+export const author = 'yosep';
+export const version = '0.1.0';
+
+const stakingAbi = [
+  'function GEN1depositsOf(address account) external view returns (uint16[] memory)'
+];
+
+const tokenAbi = [
+  'function balanceOf(address owner) public view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const stakingPool = new Multicaller(network, provider, stakingAbi, {
+    blockTag
+  });
+  const tokenPool = new Multicaller(network, provider, tokenAbi, {
+    blockTag
+  });
+
+  addresses.forEach((address) => {
+    stakingPool.call(address, options.staking, 'GEN1depositsOf', [address]);
+    tokenPool.call(address, options.token, 'balanceOf', [address]);
```

Comment:
@ekwelfweer For `balanceOf` you can use `erc20-balance-of` strategy, maybe just keep `GEN1depositsOf` in this . so the voting power will be sum of these two strategies

Reply:
hello @ChaituVR,

both tokens are erc-721. that is none of them are erc-20.

When people "stake" the ERC-721 NFT, the ownership of their nfts is transferred to the contract, and that's why we call GEN1depositsOf or balanceOf in both sides.

Reply:
oh you can use `erc721` strategy for that :) ( will call `balanceOf` function ) so i think would be great if this strategy just have `GEN1depositsOf`

Reply:
I think we can leave it like this for now :) ignore my comment